gobblin-metastore/src/main/java/org/apache/gobblin/runtime/StateStoreBasedWatermarkStorage.java

```diff
@@ -55,10 +56,10 @@ public class StateStoreBasedWatermarkStorage implements WatermarkStorage {
    * A watermark prefix that is compatible with different watermark storage implementations.
    * As such, this prefix should not include any characters disallowed in a {@link java.net.URI}.
    */
-  private static final String WATERMARK_STORAGE_PREFIX="streamingWatermarks_";
+  protected static final String WATERMARK_STORAGE_PREFIX="streamingWatermarks_";
 
   public final StateStore<CheckpointableWatermarkState> _stateStore;
-  private final String _storeName;
```

Comment:
Extensions of this class should have access to this for interacting with the state store